src/dockerhub/image-poll.ts

```diff
@@ -0,0 +1,28 @@
+import { StateManager } from '../model/manager';
+import * as Logger from '../logger';
+import { DockerHubConfiguration, DockerHubReader } from './dockerhub-reader';
+
+export type ImagePollConfiguration = DockerHubConfiguration & {};
+
+export class ImagePoll {
+    private reader: DockerHubReader;
+    private imageNamesToPoll: string[];
+
+    constructor(private state: StateManager, config: ImagePollConfiguration) {
+        this.reader = new DockerHubReader(config);
+        this.imageNamesToPoll = ['node', 'management-service'];
```

Comment:
the concerns are bit different.. the first is a list of who we want to poll

the second is who we return in the query.. it's a nuance but they're not necessarily identical. for example if the signer is ever made optional, it will appear in the first list but not the second